test-resources/java/structural_find/input/SampleEnum.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package com.uber.piranha;
+
+public enum SampleEnum {
+    FOO,
+    BAR;
+    public String getName() {
+        switch(this) {
+          case FOO:
+            return "foo";
+          case BAR:
+            return "bar";
+          }
+    }
+}
+
+public class SomeClass {
+    public void someMethod(){
+        System.out.println("Hello World!");
+    }
+}
```

Comment:
So, the reason there isn't an `expected/` output for this test case, is that this is a find rule and not any sort of transformation, correct? And our testing infra only tests the total number of matches over all tests, rather than specifically how many matches on each test or where?